tests/conftest.py

```diff
@@ -27,13 +27,13 @@ def serpent_runtime(cwd, tmpdir_factory):
     """SaltProc objects for Serpent unit tests"""
     saltproc_input = str(cwd / 'serpent_data' / 'tap_input.json')
     depcode_input, simulation_input, reactor_input = \
-        read_main_input(saltproc_input)[3]
+        read_main_input(saltproc_input)[4]
```

Comment:
Can you rewrite this in a way that doesn't use a hard-coded value?

Reply:
I can't think of any, other than using `-1` instead of `4`.